pkg/webhook/nodenetworkconfigurationpolicy/handler.go

```diff
@@ -42,3 +47,60 @@ func mutatePolicyHandler(neededMutationFor func(nmstatev1beta1.NodeNetworkConfig
 		return response
 	}
 }
+
+func mutateAllPoliciesHandler(cli client.Client, neededMutationFor func(oldNode, newNode corev1.Node) bool, mutators ...mutator) admission.HandlerFunc {
```

Comment:
Why do we need so much code for "reset the NNCP condition if node labels are changed"?

Would you please describe in a single paragraph in the commit message what is your strategy here to reset the condition?

Reply:
Expaneded commit message.